libraries/helpers.rb

```diff
@@ -432,3 +432,51 @@ def overwrite_nfs_template?
     node['platform'] == 'centos' && node['platform_version'].to_i == 8
   ].any?
 end
+
+def setup_munge_head_node
+  # Generate munge key
+  bash 'generate_munge_key' do
+    user 'root'
+    group 'root'
+    cwd '/tmp'
+    code <<-MUNGE_KEY
+      set -e
+      # Generates munge key in /etc/munge/munge.key
+      sudo -u munge /usr/sbin/mungekey --verbose
```

Comment:
why `verbose`? let's make sure this does not cause the key to be printed to stdout and end up in logs.

Reply:
The key won't be printed, it just prints out some info:
```
# With verbose
$ sudo -u munge /usr/sbin/mungekey --verbose
mungekey: Info: Created "/etc/munge/munge.key" with 1024-bit key
$

# Without verbose
$ sudo -u munge /usr/sbin/mungekey
$
```